CppUnit/include/CppUnit/Test.h

```diff
@@ -49,37 +49,8 @@ class CppUnit_API Test
 	std::vector<std::string> _setup;
 };
 
-
 inline Test::~Test() = default;
 
-
```

Comment:
are we sure these are not called from anywhere? we may end up with pure virtual calls if they are

Reply:
You are right. It seemed to me that those functions were left there by mistake. I'll revert this.